src/screens/ChatScreen/SignedChatScreen.tsx

```diff
@@ -19,21 +19,14 @@ import {setChannel} from '../../context/actions/setChannel';
 import {styles} from './AnonymousChatScreen';
 
 const SignedChatScreen = () => {
-  const {
-    selectedChannel,
-    chats,
-    goBackFromChatScreen,
-    goToChatInfoScreen,
-    sendChat,
-    updateChatContinuity
-  } = useChatScreenHook(SIGNED);
+  const {selectedChannel, chats, goBackFromChatScreen, goToChatInfoScreen, sendChat} =
+    useChatScreenHook(SIGNED);
 
   const flatlistRef = React.useRef<FlatList>();
   const [loading, setLoading] = React.useState(false);
   const [isAnonimityEnabled, setIsAnonimityEnabled] = React.useState(true);
   const [, dispatchChannel] = (React.useContext(Context) as unknown as any).channel;
   const [profile] = (React.useContext(Context) as unknown as any).profile;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `updateChatContinuity` function from the `useChatScreenHook` hook is no longer being used. If this function was important for maintaining chat continuity, its removal could lead to issues. Please ensure that chat continuity is still maintained after these changes.

```diff
- const {
-   selectedChannel,
-   chats,
-   goBackFromChatScreen,
-   goToChatInfoScreen,
-   sendChat,
-   updateChatContinuity
- } = useChatScreenHook(SIGNED);
+ const {selectedChannel, chats, goBackFromChatScreen, goToChatInfoScreen, sendChat} =
+   useChatScreenHook(SIGNED);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->